x/datadeal/genesis.go

```diff
@@ -0,0 +1,20 @@
+package datadeal
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/medibloc/panacea-core/v2/x/datadeal/keeper"
+	"github.com/medibloc/panacea-core/v2/x/datadeal/types"
+)
+
+// InitGenesis initializes the capability module's state from a provided genesis
+// state.
+func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
+	//TODO implement me
+	panic("implement me")
+}
+
+// ExportGenesis returns the capability module's exported genesis.
+func ExportGenesis(ctx sdk.Context, k keeper.Keeper) *types.GenesisState {
+	//TODO implement me
+	panic("implement me")
+}
```

Comment:
I suggest not to panic here. Because chain cannot be initialized with this panic.

Reply:
Fixed in [351464b](https://github.com/medibloc/panacea-core/pull/520/commits/351464b7b2ff484dd8d194b99a6ec565dba8e3ce).